### PR TITLE
WL-3962: Adding divs and css styles to re-position and style elements on page

### PR DESCRIPTION
**Pull request by nicholaswilson100**

### CODE REVIEW

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 1]
```diff
@@ -394,80 +461,13 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
...
-    			Schema schema = citation.getSchema();
-    			if(schema == null)	{
-    				m_log.warn("CLAS.handleViewRequest() Schema is null: " + citation);
-    				continue;
-    			}
-    			List fields = schema.getFields();
-    			Iterator fieldIt = fields.iterator();
-    			
-    			while(fieldIt.hasNext())
-    			{
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java:409]: Looks to be indentation change here.

**Comment by nicholaswilson100** [comment]
I've tried to fix it but this file is such a messy combination of tabs and spaces, it's hard to see what's right and wrong when you move lines around

File: citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java [hunk 2]
```diff
@@ -460,14 +432,42 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
...
+    			}
+    			/* not using view citation link - using toggle triangle
+    			out.println("\t\t\t\t<a id=\"link_" + escapedId + "\" href=\"#\" onclick=\"viewFullCitation('" + escapedId + "'); return false;\">"
+    					+ rb.getString( "action.view" ) + "</a>" );
+    			*/
+    			// TODO This doesn't need any Inline HTTP Transport.
+    			out.println("\t\t\t\t<span class=\"Z3988\" title=\""+ citation.getOpenurlParameters().substring(1).replace("&", "&amp;")+ "\"></span>");
+    			out.println("\t\t\t</div></div>");
+
+			    out.println("\t\t\t<div><table id=\"specialKeys\" class=\"listHier lines nolines\" cellpadding=\"0\" cellspacing=\"0\">");
```
Comment (buckett) [inline citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java]: id attribute should be unique on the page

**Comment by buckett** [comment]
I think it's got worse, the indentation doesn't match what was there previously and it is no longer indented the same.

It makes it very difficult to tell what has changed from a review point of view.

**Comment by nicholaswilson100** [comment]
Ok I'll give it another go. 

I might break it up into separate chunks as it doesn't look okay when I either leave the indentation as it was or try to change it so that it looks better and more consistent. It'll make reviewing it easier too.

**Pull request closed without merging**